src/ps2.js

```diff
@@ -106,8 +115,13 @@ function PS2(cpu, bus)
 
     this.bus.register("mouse-wheel", function(data)
     {
-        // TODO: Mouse Wheel
-        // http://www.computer-engineering.org/ps2mouse/
+        this.wheel_movement -= data[0];
+        this.wheel_movement -= data[1] * 2; // X Wheel Movement
```

Comment:
Why `data[1] * 2`? Shouldn't `data[1]` be ignored if we don't implement x wheel support?

Reply:
(answering my own question: Yes, this is how it's specified to work)